src/hooks/use-locale-number.ts

```diff
@@ -0,0 +1,42 @@
+import { useContext } from 'react'
+import { ObjectListContext } from '../context'
+import type { NumberToLocale, NumberToLocaleParameters } from '../types'
+
+export default function useLocaleNumber(params?: NumberToLocaleParameters): NumberToLocale {
+  const context = useContext(ObjectListContext)
+
+  const configuration = {
+    currency: false,
+
+    // add override from context if provided.
+    ...context,
+
+    // add override if params are provided.
+    ...params,
```

Comment:
I think you've done a good enough job making the code explain this for you :)
```suggestion
    ...context,
    ...params,
```